packages/core/test/insured-bridge/ArbitrumMessenger.js

```diff
@@ -81,7 +81,22 @@ describe("ArbitrumMessenger integration with BridgeAdmin", () => {
       from: owner,
     });
   });
+  describe("relayMessage only callable by owner", function () {
```

Comment:
small nit: do you really need both the `describe` and `it` here? fine to leave it as is just it's a bit redundant seeing there is only one test in the `describe`.

Reply:
removed